src/index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    say(name = 'world') {
+        return `Hello ${name}!`;
+    }
```

Comment:
The implementation is fine, just a little style issue: https://en.wikipedia.org/wiki/Indentation_style#Allman_style

```suggestion
    say(name = 'world')
    {
        return `Hello ${name}!`;
    }
```

Reply:
Solved with latest commit